src/test/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtilsUnitTest.java

```diff
@@ -1483,71 +1483,7 @@ public void remove() {
             }
         };
     }
-
-    @Test(dataProvider="overlapWithData")
-    public void testOverlapsWith(final VariantContext vc, final GenomeLoc genomeLoc) {
```

Comment:
Instead of deleting this test case, can you instead convert it into test coverage on the remaining `GATKVariantContextUtils.overlapsRegion(VariantContext, SimpleInterval)` method?

Reply:
Could I instead get rid of that method and replace it with the equivalent `Locatable.overlaps(Locatable)`?  They are equivalent:

The `GATKVariantContextUtils` version is
```
if (variantContext.getEnd() < region.getStart())
    return false;
if (variantContext.getStart() > region.getEnd())
    return false;
return variantContext.getContig().equals(region.getContig());
```
i.e.
```
return sameContig && vcDoesntEndBeforeRegionStarts && vcDoesntStartAfterRegionEnds;
```
The Locatable method is (I'm inlining `withinDistanceOf`, `contigsMatch`, and `CoordMath.overlaps` and expanding `Objects.equal` to delegate to `String.equals`):
```
return vc.getContig().equals(region.getContig()) && 
(region.getStart() >= vc.getStart() && region.getStart() <= vc.getEnd()) || 
(region.getEnd() >=vc.getStart() && region.getEnd() <= vc.getEnd()) ||
vc.getStart() >= region.getStart() && vc.getEnd() <= region.getEnd());
```
i.e.
```
return sameContig && 
(regionStartsInVariant || regionEndsInVariant || variantStartsAndEndsInRegion);
```
You can satisfy yourself that these are equivalent.

Other than that, the `Locatable` version returns `false` if an argument is `null`, while the `GATKVariantContextUtils` version throws an error.  But that cant be much of an obstacle.